acra/configuring-maintaining/logging/_index.md

```diff
@@ -1,4 +1,42 @@
 ---
-title: logging
+
+weight: 1
+title: Logging
 bookCollapseSection: true
 ---
+
+# Logging in Acra
+
+Starting with version [`0.77.0`](https://github.com/cossacklabs/acra/releases/tag/0.77.0), Acra supports [multiple logging formats](formats/):
+
+- PlainText
+- CEF
+- JSON
```

Comment:
Using CamelCase for PlainText is a bit too much, IMO.

There is a common word “plaintext”, but there are no ”cef” and “json”, which are actually abbreviations.